signac/core/json.py

```diff
@@ -16,6 +16,11 @@
 try:
     import rapidjson as json
     from rapidjson import Encoder
+    try:
+        # Defined for rapidjson >= 0.7.1
+        from rapidjson import JSONDecodeError
+    except ImportError:
+        JSONDecodeError = ValueError
```

Comment:
Before 0.7.1, the error thrown was a ValueError. This should ensure that the error in `contrib.filterparse` still gets caught.